cpp/src/phonenumbers/normalize_utf8.h

```diff
@@ -27,6 +27,8 @@ struct NormalizeUTF8 {
     string normalized;
     UnicodeText number_as_unicode;
     number_as_unicode.PointToUTF8(number.data(), static_cast<int>(number.size()));
+    if (!number_as_unicode.UTF8WasValid())
+      return normalized;
```

Comment:
Maybe add a comment where normalized is declared (or here) to state that it is initialized to an empty string and this value is intended to be returned here. Also, do we have a test case covering that a non-utf8 string will be normalized to empty string?

Reply:
Done